connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -193,10 +193,6 @@ public PluginClassLoader pluginClassLoader(String name) {
                : null;
     }
 
-    public ClassLoader connectorLoader(Connector connector) {
-        return connectorLoader(connector.getClass().getName());
-    }
-
     public ClassLoader connectorLoader(String connectorClassOrAlias) {
```

Comment:
Looks like this method is only invoked from `Plugins::connectorLoader`; should we make this package private in order to keep the API clean and make that method the only entrypoint?